### ISSUE TITLE
WCAG 2.1: Reflow - Long app/element names in the right pane in "UI automation tree view"are truncated at 200% zoom

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#770]
**Describe the bug**
Long app/element names in the right pane in "UI automation tree view" are not completely accessible when the pane splitter is in default position (Considering the splitter position in Automated Checks as default)

**To Reproduce**
Adjust the following display settings in Windows :
- Display resolution to ```1280 * 1024```
- Scaling size to ```200%```

1. Launch AI-Windows
2. Run test on any app
3. Navigate to "View results in UI Automation Tree"
4. Set the pane splitter to a similar position as in automated checks
5. Observe that the complete app/element name on the right pane is not available

**Expected behavior**
All content should be made available i.e. completely visible or be revealed via accessible controls

**Actual behavior**
Element name in the right pane is hidden

**Screenshots or .GIF**
![6](https://user-images.githubusercontent.com/22068434/80001634-6fcc8000-8473-11ea-9556-334e51eada99.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Enterprise, Version1903
 - Accessibility Insights for Windows Version: Production Release: v1.1.1198.01

**Comment by adarshrema** [comment]
From internal testing for accessibility. To be combined with results from planned HC testing.

**Comment by juchampa** [comment]
@iamrafan this item is not in scope
